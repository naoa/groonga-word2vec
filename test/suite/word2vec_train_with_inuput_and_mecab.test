register word2vec/word2vec

table_create Entries TABLE_NO_KEY
column_create Entries title COLUMN_SCALAR ShortText
column_create Entries content COLUMN_SCALAR ShortText

table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content

load --table Entries
[
{"title": "groonga and MySQL", "content": "groonga + MySQL = mroonga."}
]

word2vec_train Entries title --input_filter "a" --mecab_option "-F%m:%f[0]\\s -E\\0"


