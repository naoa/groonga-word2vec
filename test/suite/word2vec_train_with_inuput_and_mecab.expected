register word2vec/word2vec
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries title COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Tokens TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
[[0,0.0,0.0],true]
column_create Tokens entries_title COLUMN_INDEX|WITH_POSITION Entries title
[[0,0.0,0.0],true]
column_create Tokens entries_content COLUMN_INDEX|WITH_POSITION Entries content
[[0,0.0,0.0],true]
load --table Entries
[
{"title": "groonga and MySQL", "content": "groonga + MySQL = mroonga."}
]
[[0,0.0,0.0],1]
word2vec_train Entries title --input_filter "a" --mecab_option "-F%m:%f[0]\\s -E\\0"
